app/controllers/feeds_controller.rb

```diff
@@ -2,6 +2,37 @@ class FeedsController < ApplicationController
 
   def index
     @user = User.find_by(id: session[:user_id])
-    
+    if @user && @user.instagrams
+      @response = []
+      @user.instagrams.each do |gram|
+        @response << HTTParty.get(INSTAGRAM_URI + "#{gram.provider_id}/media/recent?access_token=#{session[:access_token]}")
+      end
+    end
+
+    if @user && @user.tweets
+      @people = []
+      # @people << Instagram.find(@user.instagram_ids)
+      @people << Tweet.find(@user.tweet_ids)
+      @people.flatten!
+      @people.sort_by! { |person| person.username.downcase }
+
+      @feed = []
+      @people.each do |person|
```

Comment:
A brief review of your code shows that you're not using this `@people` variable in the index view (though you are using a `@people` in the `people` action). So, first thought is that this does not need to be an instance variable. Second that is that the `sort_by!` block on line 17 is unnecessary, as you sort again on line 24. The sort on `@feed` makes sense to me as that's what you're outputting on the page. Sorting can be really expensive, so please take a moment to figure out if it's really necessary.